examples/app-setup.sh

```diff
@@ -11,22 +11,10 @@ DIR=`dirname "$0"`
 cd $DIR
 export FLASK_APP=app.py
 
-# Setup app
-mkdir $DIR/instance
-
-mkdir static
-
-# Install specific dependencies
-pip install -r requirements.txt
```

Comment:
I think you need to leave this one

Reply:
I have a branch to test it on my travis where I have reverted these ones.